docs/docs/cmd/planner/plan/plan-set.md

```diff
@@ -0,0 +1,48 @@
+# planner plan set
+
+Update title of a specified plan
+
+## Usage
+
+```sh
+m365 planner plan set [options]
+```
+
+## Options
+
+`-i,--id [id]`
+: ID of the plan. Specify either `id` or `title` but not both.
+
+`-t,--title [title]`
+: Title of the plan. Specify either `id` or `title` but not both.
```

Comment:
Let's include the remark that if you specify `title`, you also need to specify owner group name or -id.